packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -2,17 +2,44 @@
  * CSS Custom Properties
  *
  * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-panel-content-space: Specifies the padding of the component's content.
- * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+
+ * @prop --calcite-panel-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-panel-shadow: Specifies the shadow of the component.
+ * @prop --calcite-panel-heading-text-color: Specifies the text color of the component's `heading`.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's `description`.
+
  * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-header-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-content-top-background-color: Specifies the border color of the `"content-top"` slot.
+ * @prop --calcite-panel-content-bottom-background-color: Specifies the border color of the `"content-bottom"` slot.
+
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-header-border-color: Specifies the border color of the component's header.
+ * @prop --calcite-panel-footer-border-color: Specifies the border color of the component's footer.
+ * @prop --calcite-panel-content-top-border-color: Specifies the border color of the `"content-top"` slot.
+ * @prop --calcite-panel-content-bottom-border-color: Specifies the border color of the `"content-bottom"` slot.
+ * @prop --calcite-panel-action-bar-border-color: Specifies the border color of the `"action-bar"` slot.
+
+ * @prop --calcite-panel-space: Specifies the padding of the component's `"unnamed (default)"` slot.
```

Comment:
We need this to retain the behavior of Dialog, which sets this to a non-0 default. We could make it internal? cc @driskull (for Sheet implementation as well).

Reply:
Yes, we need this for sheet.

Reply:
an internal could make sense here.

Reply:
We have a public property currently https://developers.arcgis.com/calcite-design-system/components/panel/#api-reference-styles---calcite-panel-content-space - this would just be the replacement for that.